### ISSUE TITLE
Report Confidence Score

### ISSUE THREAD

**Issue by ktmeaton** [DerrickWood/kraken2#77]
Hello,

Would it be possible to have the final confidence score reported in the output for each read? I'm exploring the use of this metric in calculating summary statistics for classified taxa (score distribution, mean, SD, etc.). While I can calculate this myself from the kmer mapping string + NCBI's taxdump, it is much much slower than Kraken2's implementation of the taxonomy search.

Thanks,
Katherine

**Comment by ciemanek** [comment]
This would be very helpful indeed!

**Comment by Midnighter** [comment]
I agree it would be nice but there is a separate tool if you really want the information. https://github.com/Ivarz/Conifer

**Comment by JuliaMcGonigle** [comment]
Is the --confidence flag in kraken2 doing the same thing as Conifer?

**Comment by Midnighter** [comment]
No, that flag is setting a threshold on the confidence score. Conifer is reporting the confidence score.
